test-network-function/lifecycle/suite.go

```diff
@@ -322,7 +356,7 @@ func shutdownTest(podNamespace, podName string) {
 func cleanupNodeDrain(env *config.TestEnvironment, nodeName string) {
 	uncordonNode(nodeName)
 	for _, ns := range env.NameSpacesUnderTest {
-		notReady := waitForAllDeploymentsReady(ns, scalingTimeout, scalingPollingPeriod)
+		notReady := waitForAllDeploymentsReady(ns, scalingTimeout, scalingPollingPeriod, "deployment")
```

Comment:
Shouldn't we include here an extra waitForAllPodSets() check for "statefulsets" after the "deployments" one?